packages/alfa-rules/test/sia-r2/rule.spec.tsx

```diff
@@ -32,20 +32,27 @@ test("evaluate() fails an <img> element without an accessible name", async (t) =
   ]);
 });
 
-test("evaluate() is inapplicable to a document without images", async (t) => {
-  const document = Document.empty();
+test("evaluate() is inapplicable to an <img> element with a presentational role", async (t) => {
+  const document = Document.of([<img role="none"></img>]);
 
   t.deepEqual(await evaluate(R2, { document }), [inapplicable(R2)]);
 });
 
-test("evaluate() is inapplicable to an image that is marked as decorative", async (t) => {
-  const document = Document.of([<img role="none"></img>]);
+test(`evaluate() is inapplicable to a non-draggable <img> element with a
+      presentational role`, async (t) => {
+  const document = Document.of([<img role="none" draggable="false"></img>]);
```

Comment:
How is `draggable="false"` affecting the rule? Given that these two tests give the same result whether it's here or not…
If we want to illustrate `draggable` in the rule there should be a test where it does make a difference to have it.

Reply:
The point is precisely that it _shouldn't_ affect the result, as per #444. `<img draggable="false">` was previously considered focusable, meaning you couldn't assign it a presentational role.

Reply:
All add a reference to the issue in a comment.

Reply:
😅 
🤔 shouldn't this test rather be in `alfa-dom` as a test for `isSuggestedFocusable`? Since the error was there rather than in the rule itself.

Reply:
Yes and no; there are some tests for `Element#tabIndex()` is `alfa-dom`, but the test case in `alfa-rules` is really there to document a prior bug and prevent regression.